clappr/clappr-ios
# Create Time Indicator Plugin

### Goal

Creating a TimeIndicator plugin to append in MediaControl plugins.

### How to test

Run example and tests: 

Execute the video and make sure that you can see the time indicator on the left bottom of the media control. Try to execute multiples scenarios as seek a playing video, seek a paused video, enter and exit fullscreen and so on.
